packages/@n8n/nodes-langchain/nodes/trigger/ChatTrigger/constants.ts

```diff
@@ -0,0 +1,122 @@
+// CSS Variables are defined in `@n8n/chat/src/css/_tokens.scss`
```

Comment:
If we had a build system like esbuild/vite in nodes packages we could import these as raw text and we wouldn't need the duplication 😞 But it's using tsc directly so we can't do it for now

Reply:
Maybe someday :D